wordstats/edit_distance_translate.py

```diff
@@ -0,0 +1,36 @@
+import codecs
+import configparser
+from .cognate_files_path import *
+from .edit_distance_function_factory import WordDistanceFactory
+from googletrans import Translator
+
+class WordDistanceOverlap(WordDistanceFactory):
```

Comment:
`WordDistanceOverlap`, a kind of `WordDistance`, can't be the subclass of a `WordDistanceFactory`. 
This makes me think that your superclass should be actually named `WordDistance`. 
Would that make sense to you?